samples/account-linking/csharp/Source/AccountLinking.Service/Services/AccountLinkingStateService/AccountLinkingConfiguration.cs

```diff
@@ -0,0 +1,8 @@
+using System.Text.Json;
+
+namespace Microsoft.Teams.Samples.AccountLinking.Service.State;
+
+public class AccountLinkingConfiguration
+{
+    public IDictionary<string, JsonElement> ConnectionConfigurations {get;set;} = new Dictionary<string, JsonElement>();
```

Comment:
Is JsonElement the right type here? it couples you to a JSON-based implementation.